tests/api_gov.spec.js

```diff
@@ -256,6 +256,27 @@ describe('TESTING /v2/gov/austria', () => {
             });
     });
 });
+describe('TESTING /v2/gov/vietnam', () => {
+    it('/v2/gov/vietnam correct fields set', (done) => {
+        chai.request(app)
+            .get('/v2/gov/vietnam')
+            .end((err, res) => {
+                should.not.exist(err);
+                should.exist(res);
+                res.should.have.status(200);
+                res.body.should.be.a('array');
+                res.body.forEach((element) => {
+                    element.should.have.property('updated');
+                    element.should.have.property('city');
+                    element.should.have.property('cases');
+                    element.should.have.property('beingtreated');
```

Comment:
this should be 'beingTreated' (also only works if you make the properties camelcase not titlecase..